src/main/java/frc/robot/commands/notSwerve/RejectBall.java

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.notSwerve;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.RobotContainer;
+import edu.wpi.first.wpilibj.util.Color;
+import frc.robot.subsystems.ColorSensor;
+import frc.robot.Robot;
+
+public class RejectBall extends CommandBase {
+  /** Creates a new RejectBall. */
+
+  Color color;
+
+  public RejectBall() {
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(RobotContainer.colorSensor, RobotContainer.intake);
+  }
+
+  // You thought that this comment would explain what initialize does, but it was me, DIO!
+  @Override
+  public void initialize() {
+    //spin the intake motors
+
+    //Todo set value for ball color
```

Comment:
This logic feels very odd.
Properly, we should be writing our own trigger (https://docs.wpilib.org/en/stable/docs/software/commandbased/binding-commands-to-triggers.html#creating-your-own-custom-trigger) and then running the intake command backwards based off of that trigger.